LDAR_Sim/external_sensors/METEC_NO_WIND.py

```diff
@@ -0,0 +1,149 @@
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+import numpy as np
+from utils.attribution import update_tag
+from methods.funcs import measured_rate
+
+
+def detect_emissions(self, site, covered_leaks, covered_equipment_rates, covered_site_rate,
+                     site_rate, venting, equipment_rates):  # pylint: disable=unused-argument
+    """ 
+    An alternative sensor, specifically built to replicate the probability of detection
+    curves provided by a METEC report, which does not factor in wind speeds
+
+    Utilizes 3 values set as the MDL:
+        mdl = [a, b, c]
+        where 
+        a and b : utilize for the PoD curve variables 
+        c : represents the floor/minimum cutoff value of the leak rates that the sensor can detect
+
+    PoD = 1 / (1 + e ^ (a - b * r )) 
+
+    a = first MDL value
+    b = second MDL value
+    r = emission rate
+    W = wind speed
+
+    Args:
+        site (site obj): Site in which crew is working at
+        covered_leaks (list): list of leak objects that can be detected by the crew
+        covered_equipment_rates (list): list of equipment leak rates that can be
+                                        detected by the crew
+        covered_site_rate (float): total site emissions from leaks that are observable
+                                    from a crew
+        site_rate (float): total site emissions from leaks all leaks at site
+        venting (float): total site emissions from venting
+        equipment_rates (list): list of equipment leak rates for each equipment group
+
+    Returns:
+        site report (dict):
+                site  (site obj): same as input
+                leaks_present (list): same as covered leaks input
+                site_true_rate (float): same as site_rate
+                site_measured_rate (float): total emis from all leaks measured
+                equip_measured_rates (list): total of all leaks measured for each equip group
+                venting (float): same as input
+                found_leak (boolean): Did the crew find at least one leak at the site
+
+
+    """
+
+    missed_leaks_str = '{}_missed_leaks'.format(self.config['label'])
+    equip_measured_rates = []
+    site_measured_rate = 0
+    found_leak = False
+    n_leaks = len(covered_leaks)
+    mdl = self.config['sensor']['MDL']
+
+    if self.config["measurement_scale"] == "site":
+        # factor of 3.6 converts g/s to kg/h
```

Comment:
Can documentation be added to the top of the file about expected and returned units? If other parties are going to make their own sensors it would be helpful for them to have this clearly laid out.

Reply:
Added note to the read me instead.